src/emulator/stringtable.c

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file stringtable.c
+ *
+ * This file implements tools to read a string table (following the ST10 format)
```

Comment:
do you have a link for "ST10"? couldn't find much by googling

Reply:
yeah I didn't find anything either, I think it's a custom format, ST10 could mean "String Table v1.0" or something

Reply:
maybe put it in quotes to make it clear that the reader is not expected to know this?